cross/libcaca/Makefile

```diff
@@ -12,6 +12,11 @@ COMMENT  = The industry-standard colour ASCII-art library.
 LICENSE  = WTFPL
 
 GNU_CONFIGURE = 1
+PRE_CONFIGURE_TARGET = run_boostrap
```

Comment:
Please prefix with module name. For homogeneity, I would have called that target "libcaca_pre_configure"